catalog/ui/catalog-ui-search/src/main/webapp/component/workspace-lists/workspace-lists.hbs

```diff
@@ -0,0 +1,31 @@
+{{!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ --}}
+<div align="center" class="lists-empty">
+    <div class="is-header">
+        You don't have any lists.  <button class="quick-search is-primary in-text">Search</button> 
+        for something and add it to a list or create a 
+        <div class="quick-create is-primary in-text composed-button">
```

Comment:
It's because that div is populated with a `PopoutView.createSimpleDropdown` that needs to be passed the text to display as a parameter
https://github.com/codice/ddf/pull/2886/files/eb4c5fd615daf78bdd4ed2ddf87daae7974e9804#diff-1cdf0ea194aec512d1275398e6544c45R57